Makefile

```diff
@@ -0,0 +1,28 @@
+mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
+current_dir := $(dir $(mkfile_path))
+BUILDDIR := $(current_dir)_build
+
+all: prep fetch-data create-html
+
+prep:
+	@echo "Preparing build directory..."
+	rm -rf $(BUILDDIR)
+	mkdir -p $(BUILDDIR)
+	mkdir -p $(BUILDDIR)/data
+	mkdir -p $(BUILDDIR)/plugins
+	mkdir -p $(BUILDDIR)/static
+	cp -r $(current_dir)templates $(BUILDDIR)/templates
+	cp -r $(current_dir)static/images $(BUILDDIR)/static/images
+	cp -r $(current_dir)static/css $(BUILDDIR)/static/css
+	cp $(current_dir)entire_text_search.js $(BUILDDIR)/entire_text_search.js
+	cp $(current_dir)index.html $(BUILDDIR)/index.html
+
+fetch-data:
+	@echo "Fetching data..."
+	python $(current_dir)fetch_napari_data.py $(BUILDDIR)
+	@echo "Data fetched and stored in $(BUILDDIR)/data"
+
+create-html:
+	@echo "Creating HTML files..."
+	python $(current_dir)create_static_html_files.py $(BUILDDIR)
+	@echo "HTML files created in $(BUILDDIR)/plugins"
```

Comment:
I think you should indicate all of these targets as ["phony"](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html), because the steps are not generating output files named for their targets.

```
.PHONY: all prep fetch-data create-html
```

I think the logic for finding `current_dir` is nice, but possibly unnecessary? I think it's only needed if running `make -f path/to/Makefile` instead of just `make` from the project root, so I'd just vote to remove it and use relative paths. I also found it a little confusing that `current_dir` has a trailing slash while `BUILDDIR` does not.

All that said...if none of us are super familiar with `make` I'd be in favor of writing this script in Python instead.

All _that_ said, this seems fine to me as-is!

Reply:
I agree with PHONY remark

Reply:
Ahh thanks for this review @aganders3 this is all new to me so appreciate the info. I will update the target and use relative paths for the directories. So I can iterate more quickly (and because of the looming deadline :sweat_smile:) I'll keep the makefile for now, but we can switch to a python script down the line if we think the makefile isn't our cup of tea, and I'll make an issue for that.